components/button/button-icon.js

```diff
@@ -126,9 +128,9 @@ class ButtonIcon extends ButtonMixin(VisibleOnAncestorMixin(RtlMixin(LitElement)
 				?formnovalidate="${this.formnovalidate}"
 				formtarget="${ifDefined(this.formtarget)}"
 				name="${ifDefined(this.name)}"
-				title="${ifDefined(this.text)}"
 				type="${this._getType()}">
 				<d2l-icon icon="${ifDefined(this.icon)}" class="d2l-button-icon"></d2l-icon>
+				${this.text ? html`<d2l-tooltip for="target" disable-focus-lock>${this.text}</d2l-tooltip>` : null}
```

Comment:
The tooltip needs to be inside the `button`. If it isn't then the `offsetParent` changes from the `d2l-button-icon` to a different element when the VOA transition finishes causing it to be positioned incorrectly. Because the `button` is positioned `relative`, this doesn't happen. I chose this rather than positioning the `d2l-button-icon` itself `relative`.